packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,115 @@
+In order to specify a resource targeted by a resource set, you will simply use the REST URL of the corresponding Okta API:
+* [All users](/docs/reference/api/users/#list-users)
+  ``` http
+  https://${yourOktaDomain}/api/v1/users
+  ```
+* [All groups](/docs/reference/api/groups/#list-groups)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups
+  ```
+* [A specific group](/docs/reference/api/groups/#get-group)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}
+  ```
+* [All users within a specific group](/docs/reference/api/groups/#list-group-members)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users
+  ```
+
+> **Note:** All users and groups could be represented using either of [All users](/docs/reference/api/users/#list-users) or [All groups](/docs/reference/api/groups/#list-groups) links. There is no functional difference between the two.
+
+> **Note:** If you use a Role with permissions that don't apply to the resources in the resource set, the admin role will have no effect. For example, the `okta.users.profile.manage` permission will give the admin no privileges if granted to a resource set that only included `https://${yourOktaDomain}/api/v1/groups/${targetGroupId}` resources. If you want the admin to be able to manage the users within the group the resource set must include the corresponding `https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users` resource.
+
+#### Binding Member Identifiers
+In order to specify binding members, you use the REST URL of the corresponding Okta API:
+* [A specific user](/docs/reference/api/users/#get-user)
+  ``` http
+  https://${yourOktaDomain}/api/v1/users/${memberUserId}
+  ```
+* [A specific group](/docs/reference/api/groups/#get-group)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups/${memberGroupId}
+  ```
+
+## Custom vs. Standard
+1. An admin can have both standard role assignments and custom role bindings. Note that privileges granted to an admin are an aggregate of:
+    * standard roles directly assigned
+    * standard roles granted through group membership
+    * custom roles directly assigned
+    * custom roles granted through group membership
+
+    As a result, if an admin was granted a standard role limited to a single group and at the same time received group management privileges on all groups in the org through a custom role, the ultimate outcome is group management on all groups.
+2. During BETA, Custom Roles, Permissions and Resources are limited to User and Group related items. We'll be introducing additional Resources and Permissions over time and appreciate your feedback.
+3. A Custom Role cannot be assigned without a Resource Set, hence always being applicable only to a subset of resources. Standard roles on the other hand are always initially granted at the entire org. They are only scoped to specific resources by subsequent invoking of the [target operations](/docs/reference/api/roles/#role-target-operations).
```

Comment:
custom role > lowercase
resource-set > lowercase and hyphenated

Standard roles, on the other hand, are... > commas between "on the other hand"